packages/web-components/src/_docs/concepts/introduction.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta } from '@storybook/addon-docs';
+import pkg from '../../../package.json';
+
+<Meta title="Concepts/Introduction" />
+
+<h2>
+  Fluent UI Web Components <span>v{pkg.version}</span>
+</h2>
+
+⚠️ **While in Beta, breaking changes may be made based on feedback from users.** This space is useful for testing new components whose APIs might change before final release.
+
+Microsoft's [Fluent UI Web Components](https://github.com/microsoft/fluentui/tree/master/packages/web-components) is designed to help you build Fluent web apps using extensible Web Components. The package composes the `@microsoft/fast-foundation` Web Component package and styles it with the [Fluent design language](https://github.com/microsoft/fluentui).
```

Comment:
```suggestion
Microsoft's [Fluent UI Web Components](https://github.com/microsoft/fluentui/tree/master/packages/web-components) is designed to help you build Fluent web apps using extensible Web Components. The package composes the `@microsoft/fast-foundation` Web Component package and styles it with the [Fluent design language](https://fluent2.microsoft.design/).
```

Seems like a better URL given the text.

Reply:
There's a Readme update PR that I'll pull in here to update this after the PR is merged.